From f537d427df2dbac48f3f55d37456bec656c66231 Mon Sep 17 00:00:00 2001
From: Michael S. Tsirkin <mst@redhat.com>
Date: Thu, 24 Jun 2010 20:19:02 +0300
Subject: [PATCH] make-release: fix mtime on rhel6 beta

With recent git versions (like one in rhel6 beta),
git prints out tag info in addition to
requested format, if passed a tag name.
This breaks make-release.
To get just the time as we want, we need to
pass something that is not a tag name but matches
same commit.  $commit^{commit} does this.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
---
 kvm/scripts/make-release |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kvm/scripts/make-release b/kvm/scripts/make-release
index e8051f4..64e77f9 100755
--- a/kvm/scripts/make-release
+++ b/kvm/scripts/make-release
@@ -51,7 +51,7 @@ cd "$(dirname "$0")"/../..
 mkdir -p "$(dirname "$tarball")"
 git archive --prefix="$name/" --format=tar "$commit" > "$tarball"
 
-mtime=`git show --format=%ct "$commit" --`
+mtime=`git show --format=%ct "$commit""^{commit}" --`
 tarargs="--owner=root --group=root --mtime=@$mtime"
 
 mkdir -p "$tmpdir"
-- 
1.7.0.3

